ci-operator/step-registry/assisted/baremetal/operator/gather/assisted-baremetal-operator-gather-commands.sh

```diff
@@ -53,9 +53,13 @@ fi
 
 cd "\${REPO_DIR}"
 
+cat << VARS >> /root/config
+export DISCONNECTED="${DISCONNECTED:-}"
+VARS
```

Comment:
why this is needed?
we should do ``set -x`` I think which will print everything